examples/pytorch/question-answering/run_qa.py

```diff
@@ -89,6 +98,13 @@ class DataTrainingArguments:
     Arguments pertaining to what data we are going to input our model for training and eval.
     """
 
+    recipe: Optional[str] = field(
+        default=None,
+        metadata={"help": "The input file name for the Neural Magic pruning config"},
```

Comment:
Nit change to "Path to a SparseML sparsification recipe, see https://github.com/neuralmagic/sparseml for more information"